packages/material-ui/src/TextField/TextField.spec.tsx

```diff
@@ -67,3 +67,19 @@ function FocusHandlerTest() {
 
   return null;
 }
+
+function createElementTextFieldTest() {
+  // $ExpectError
+  React.createElement<TextFieldProps>(TextField, {
+    FormHelperTextProps: {
+      component: 'some-wrong-element',
+    },
+  });
+
+  // $ExpectError
+  React.createElement<TextFieldProps>(TextField, {
```

Comment:
What about using the JSX syntax instead createElement?

Reply:
If it works, i can change it. I was looking at pr u provided above, and made similar tests. But ye, i can change to jsx.